src/main/webapp/resources/js/pages/analysis/components/jobErrors/StandardError.jsx

```diff
@@ -6,12 +6,16 @@
  */
 
 import React from "react";
-import { CodeBlock } from "../../../../components/typography";
+import { OutputWrapper } from "../../../../components/OutputFiles/OutputWrapper";
 
 export function StandardError({ galaxyJobErrors, currIndex }) {
   // Returns the standard error for the given index from the jobErrors object
   function getStandardError(index = 0) {
-    return <CodeBlock>{galaxyJobErrors[index].standardError.trim()}</CodeBlock>;
+    return (
+      <OutputWrapper overflowRequired={true} style={{ whiteSpace: "pre-wrap" }}>
```

Comment:
Same as last comment about style being internal to the component.  Obviously needs to be since this is the second place you used it.

Reply:
Updated in 02cc213ba4a9097b9f3be5383bd1a1a5f5973211